pages/home/home.sql

```diff
@@ -120,7 +120,7 @@ WITH
             JOIN pl_courses AS c ON (
                 c.id = ci.course_id
                 AND c.deleted_at IS NULL
-                AND c.example_course IS FALSE
+                -- AND c.example_course IS FALSE
```

Comment:
For discussion: I wanted to be able to simulate a user clicking on their course from the home page, but this filter prevented me from doing that (at least without hardcoding a course instance ID). I believe removing this will be safe, as we have code on the enrollment page that should prevent anyone from actually enrolling in the example course. Only my forceful backend enrollments in the authn middleware should ever be enrolling users in the example course.

Reply:
Unfortunately we have a lot of users who are already enrolled in the example course, because they accessed it as an instructor but then did "View as student". Showing XC 101 on their homepage will just be clutter and they won't be able to remove it (because of the enrollment page restrictions).

As a slightly different horrible hack, how about adding a query param to the homepage to request viewing of the example course (if we are enrolled in it). That is, we'd change the above to:
```
AND (c.example_course IS FALSE OR $show_example_course)
```

Reply:
Oh huh - "view as student" actually inserts an enrollment?

I can certainly do this with a query param, yeah.

Reply:
I implemented support for an `?include_example_course_enrollments=true` in 8aced71. It's kinda hacky, but it's a small and well-documented amount of code.